README.md

```diff
@@ -9,7 +9,7 @@ You can copy these and modify them for your own use.
 
 Install the prerequisites:
 
--   either Ansible: e.g. `pip install ansible>=2.6.16,<2.7`. Other versions of 
+-   either Ansible: e.g. `pip install ansible`. At the time of writing, the Ansible version was 2.13.2. Other versions of 
```

Comment:
Officially, I think the distinction is made as follows (https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#selecting-an-ansible-package-and-version-to-install) i.e.:
- 2.13.2 is the current latest of the minimal language library called `ansible-core`
- the corresponding version of the batteries-included package (`ansible`) including collections is 6.2.0

Note also we have no experience in deploying playbooks with this version. Our production systems and the tests of our Ansible repos are still using Ansible 2.9.x where this distinction did not exist

Reply:
``ansible [core 2.11.12] `` passed the test in https://github.com/ome/ansible-example-omero-onenode/pull/11#issuecomment-1211908122